docs/src/convert-to-onda.md

```diff
@@ -190,17 +191,96 @@ new_plan = plan_edf_to_onda_samples_groups(new_plan)
+OndaEDF.jl provides methods for `Onda.store` that accept [`ConvertedSamples`](@ref) and propagate `missing` samples.
+
+Note that `Onda.store` requires at a minimum the file path (or "path-like" writeable), the format to use to serialize the samples (like `"lpcm"` or `"lpcm.zst"`), and the `Samples` themselves.
+However, this only _serializes_ the samples to the file path.
+In most cases, you will want a _signal record_ that can be stored elsewhere, which combines the path and file format to the serialized samples along with the `SamplesInfo` and other metadata required to be able to _deserialize_ and use these samples.
+In this case, you should use the second `Onda.store` method which additionally requires:
+
+- `recording::UUID`: the recording that this signal is associated with
+- `start::Period`: the start time of this signal _relative to the start of the recording_
+- `sensor_label::AbstractString`: a _unique label_, defaults to the the `sensor_label` from the `ConvertedSamples`
+
+While OndaEDF.jl generally creates `sensor_label`s which are valid when considering the EDF file alone, Onda requires that `sensor_label` be unique for the entire _recording_.  If you have signals from other sources that are associated with the same recording (e.g., different devices, multiple EDF files, etc.) that may have common `sensor_type`s, you must ensure that the `sensor_label`s that you store the signals with are _globally_ unique.
+
+#### Example: storing samples and plan
+
+```julia
+edf = EDF.File(edf_path)
+converted_samples = edf_to_onda_samples(edf, plans)
+
+# generate SignalV2 rows
+start_relative_to_recording = Second(0)
+signals = map(converted_samples) do converted
+    return Onda.store(joinpath("samples", recording, converted.sensor_label * ".lpcm"),
+                      "lpcm", converted, recording, start_relative_to_recording)
+end
+
+# write combined signals table out
+Legolas.write(string(recording, ".onda.signal.arrow"),
+              Onda.SignalV2SchemaVersion(),
+              skipmissing(signals))
+
+# write plan table out
+Legolas.write(string(recording, ".ondaedf.file-plan.arrow"),
+              OndaEDF.FilePlanV4SchemaVersion(),
+              OndaEDF.get_plan(converted_samples))
+```
+
+#### Example: generating unique `sensor_label`s
+
+Consider a case with parallel recordings from a traditional EEG system (received as an EDF) and some custom EEG hardware (which emits Onda-formatted data natively).
+Each device generates multichannel EEG data and uses `sensor_type="eeg"`.
+We must adjust the generated sensor labels for the EDF signals before storing them so that they are unique, which we can do by prepending `"edf_"` to the default, `sensor_type`-derived labels before storing:
```

Comment:
we don't do this here! also I think we should suggest modifying the plans rather than post-conversion

Reply:
that's what these lines are doing: https://github.com/beacon-biosignals/OndaEDF.jl/blob/dd97c77d0912f20b77a5beb1e5971058f85342a7/docs/src/convert-to-onda.md?plain=1#L260-L262

Reply:
I tried to separate the planning and modifying stages to make it clearer waht was going on , and added some comments.